operators/pkg/controller/common/watches/handler.go

```diff
@@ -80,7 +83,6 @@ func (d *DynamicEnqueueRequest) RemoveHandlerForKey(key string) {
 	d.mutex.Lock()
 	defer d.mutex.Unlock()
 	delete(d.registrations, key)
-	log.V(1).Info("Removed handler registration", "removed", key, "current_registrations", d.registrations)
 }
```

Comment:
Reason for removing: addition and removal happen at every single reconciliation. Most times they are no-op.